Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.6-slim
+COPY setup.* /opt/airflow/
+COPY airflow /opt/airflow/airflow
+
+ARG AIRFLOW_HOME=/usr/local/airflow
+ARG AIRFLOW_DEPS="all"
+ARG PYTHON_DEPS=""
+ARG buildDeps="freetds-dev libkrb5-dev libsasl2-dev libssl-dev libffi-dev libpq-dev git"
+ARG APT_DEPS="$buildDeps libsasl2-dev freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
```

Comment:
Having all of these in the "official" image would make it rather heavy weight for most users wouldn't it? What is the size of this image?

Reply:
I did bring it already down to 1 Gb now, before it was 1,6 Gb because of the non slim python image.
I dit this on purpose because I like that the default image does support everything. But I did make it with ARG so that it would be possible to generate multiple images from the same docker file in case a smaller version is also required

Reply:
Oke, after fixing some of the comments it did drop to 877MB now